src/Auth/helpers.js

```diff
@@ -44,7 +44,29 @@ async function blockValue(field){
     }
   }
 
+  function getTeamid(profileObject){
+    if (typeof profileObject !== "undefined" && propertyExists(profileObject, "team")){
+        if (propertyExists(profileObject, "team")){
+        return profileObject.team.id;
+        }
+    } else {
+        return null;
+    }
+  }
+
+  function getSupervisorid(profileObject){
+    if (typeof profileObject !== "undefined" && propertyExists(profileObject, "supervisor")){
+        if (propertyExists(profileObject, "supervisor")){
```

Comment:
Repeated logic test from above.  The second logic test on line 58 is the same logic test on line 59.